### DISCUSSION TITLE
Clickable link from table and image display

### DISCUSSION THREAD

**Discussion by markplus** [yajra/laravel-datatables#3104] [Q&A]
How can I generate a url in the column containing the link to open the details page from the table? I can't even insert the image display into the table, where can I find help to do this?
I'm generating the tables with the php artisan datatables:make command and I also have the editor, I'm trying to insert a details button but I haven't figured out how yet, each row has a details page that I'd like to open but I haven't figured out how yet Do

**Comment by yajra** [comment]
Just use [rawColumns](https://yajrabox.com/docs/laravel-datatables/10.0/raw-columns#main-content) to display html contents.

**Reply by markplus** [reply]
First in the blade file, I used this block of code in the table row, but now how do I transform it to use it as you recommended? This is not yet clear to me and I can't find any documentation to understand how to do it

```
 <td>
         @if(!empty($data['image']))
               <img style="width: 120px; height: 120px;" src="{{ asset('front/images/images/'.$data['image']) }}">
                 &nbsp;|&nbsp;
              <a target="_blank" href="{{ url('front/images/images/'.$data['image']) }}">Immagine</a>
         @else
              <img style="width: 120px; height: 120px;" src="{{ asset('front/images/images/small/no-image.png') }}">
         @endif
 </td>
```

**Comment by markplus** [comment]
I got help and solved it
